scripts/test/ProcessManagement/TransitionAction/TransitionActionCheckCondition.t

```diff
@@ -0,0 +1,715 @@
+# --
+# Copyright (C) 2021 Znuny GmbH, https://znuny.org/
+# --
+# This software comes with ABSOLUTELY NO WARRANTY. For details, see
+# the enclosed file COPYING for license information (GPL). If you
+# did not receive this file, see https://www.gnu.org/licenses/gpl-3.0.txt.
```

Comment:
Hi @paulfolkers 

sorry for that:

```suggestion
# This software comes with ABSOLUTELY NO WARRANTY. For details, see
# the enclosed file COPYING for license information (AGPL). If you
# did not receive this file, see http://www.gnu.org/licenses/agpl.txt.
```

Please move this test to scripts/test/ProcessManagement/TransitionAction/Base/TransitionActionCheckCondition.t

Reply:
No problem, committed. ;-)